services/postprocessing/pkg/service/service.go

```diff
@@ -182,10 +184,17 @@ func (pps *PostprocessingService) processEvent(e events.Event) error {
 func (pps *PostprocessingService) getPP(sto store.Store, uploadID string) (*postprocessing.Postprocessing, error) {
 	recs, err := sto.Read(uploadID)
 	if err != nil {
+		if err == store.ErrNotFound {
+			return nil, ErrNotFound
+		}
 		return nil, err
 	}
 
-	if len(recs) != 1 {
+	if len(recs) == 0 {
+		return nil, ErrNotFound
+	}
+
+	if len(recs) > 1 {
```

Comment:
ok, since we are expecting a result we have to convert the empty list to an error.